java/ql/lib/ext/threatmodels/threat-model-grouping.model.yml

```diff
@@ -0,0 +1,25 @@
+extensions:
+
+  - addsTo:
+      pack: codeql/java-all
+      extensible: threatModelGrouping
+    data:
+      # Default threat model
+      - ["remote", "default"]
+      - ["uri-path", "default"]
+
+      # Android threat models
+      - ["android-widget", "android"]
```

Comment:
The `android-widget` kind was removed in https://github.com/github/codeql/pull/13136, and should probably be removed from here as well as a result.

Reply:
Uh - thank you!